PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,3 +325,13 @@ def miniAOD_customizeAllMC(process):
     miniAOD_customizeCommon(process)
     miniAOD_customizeMC(process)
     return process
+
+def miniAOD_forPhase2(process):
+    print "Switching off electron and photon IDs (not yet working for Phase2 era)"
+    process.patElectrons.addElectronID = False
+    process.patElectrons.electronIDSources = cms.PSet()
+    process.patPhotons.addPhotonID = False 
+    process.patPhotons.photonIDSources = cms.PSet()
```

Comment:
Just to be clear: I want all the edits here (4 of them only : addElectronID addPhotonID, modifyElectrons, and modifyPhotons) be moved to their respective cfi files.
None of these are changed by the fact that these are miniAOD or PAT.

and the setting of electronIDSources and photonIDSources does not change the behavior and can be removed (after the boolean switches are applied)